src/pages/bezoek/[slug]/index.tsx

```diff
@@ -108,6 +118,18 @@ const VisitPage: NextPage<VisitPageProps> = () => {
 	 */
 
 	const renderPageContent = () => {
+		if (showNoAccess) {
```

Comment:
isn't it safer to just use the boolean values here, instead of storing them in a useState? Then we're sure they are always updated correctly when isLoggedIn or hasAccess changes
```suggestion
		if (!isLoggedIn || !hasAccess) {
```